code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/simpletabbox/SimpleTabBoxForm.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {
+  Action, CheckBoxField, Event, Form, FormModel, GroupBox, InitModelOf, Menu, MessageBoxes, models, PropertyChangeEvent, scout, SimpleTabAreaDisplayStyle, SimpleTabAreaPosition, SimpleTabBox, SimpleTabBoxViewActivateEvent, SimpleTabView,
+  SmartField, Status, StatusSeverity, StringField
+} from '@eclipse-scout/core';
+import SimpleTabBoxFormModel from './SimpleTabBoxFormModel';
+import {SimpleTabBoxFormWidgetMap} from '../index';
+
+export class SimpleTabBoxForm extends Form {
+  declare widgetMap: SimpleTabBoxFormWidgetMap;
+
+  simpleTabBox: SimpleTabBox;
+  addTabMenu: Menu;
+  deleteTabMenu: Menu;
+
+  simpleTabAreaPropertiesBox: GroupBox;
+  displayStyleField: SmartField<SimpleTabAreaDisplayStyle>;
+  positionField: SmartField<SimpleTabAreaPosition>;
+
+  simpleTabPropertiesBox: GroupBox;
+  titleField: StringField;
+  subTitleField: StringField;
+  iconIdField: SmartField<string>;
+  closableField: CheckBoxField;
+  statusField: SmartField<StatusSeverity>;
+
+  protected _tabCounter: number;
```

Comment:
There is no need to separate the declaration and the initialization. As there is no difference in e.g. the `simpleTabBox` being `undefined` or `null`, the majority of the constructor can be removed.
The `_tabCounter` initialization can be replaced by
```suggestion
  protected _tabCounter = 0;
```